### ISSUE TITLE
Missing newlines in webpages using empty paragraph tags

### ISSUE THREAD

**Issue by ajirving** [nvaccess/nvda#20369]
### Brief summary

I've noticed that on some webpages NVDA does not insert a newline even though one occurs on the page.  This seems to be caused by pages using an empty paragraph tag `<p></p>` to insert whitespace.

### Steps to reproduce

Go to https://foswiki.org/System/BeginnersStartHere
Look for the line starting "The basic function of Foswiki is a Wiki".  Visually the page has a newline before "A Wiki is like a web site," but NVDA doesn't show that, in fact there isn't even a space before the start of that sentence.  The cause seems to be the use of `<p></p>` to inser whitespace which I can demonstrate in a simple test HTML page.

### Actual behavior

The 2 lines on the page are presented by NVDA concatinated.

### Expected behavior

NVDA should have a newline in its rendering of the page.

### NVDA type

installed copy

### NVDA version

2026.1

### Have you tried any other versions of NVDA? If so, please report their behaviors.

Not rigorously but I believe this has been an issue for some time

### Windows version

Windows 11

### Name and version of other software in use when reproducing the issue

Same behaviour in Chrome and Edge

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by ajirving** [comment]
[test.html](https://github.com/user-attachments/files/29162358/test.html) is a simple page demonstrating what I mean.  Windows Narrator correctly renders the newline.  The use of empty paragraphs to add whitespace is probably considered bad HTML usage but ideally NVDA would render it the same as browsers do rather than concatinating paragraphs.

**Comment by seanbudd** [comment]
Closing as won't fix, this is intended behavour. Can you clarify why a blind screen reader user would need to hear such noisy whitespace?

**Comment by ajirving** [comment]
I'm not saying that there should be a blank line, just a linebreak where this occurs. Otherwise you get two lines joined together without even a space between them. I think the issue here is that the paragraph's either side of this are not wrapped in p tags.  Foswiki seems to generate pages like this and its really annoying to have all the paragraphs joined together into a single block without any linebreaks.
